lib/phlex/context.rb

```diff
@@ -76,7 +76,17 @@ def _attributes(attributes)
 
       buffer = first_render ? buffer = +"" : buffer = @_target
 
+      attributes.keys.each do |key|
```

Comment:
if you want ROFLscale
```suggestion
      attributes.each_key do |key|
```

Saves an intermediate array.

Reply:
Why not‽